### PR TITLE
ui: Create PopoverSelect, CatalogToolbar, and update tests

### PR DESCRIPTION
**Pull request by kaxcode**
-  Add rule to print 'ember/no-global-jquery' as a warning
-  Create tests for sorting services with CatalogToolbar
-  ui: Adds `selectable-key-values` helper (#7472)
-  Create CatalogToolbar component and Styling
-  Create PopoverSelect component and styling

co-authored by: @johncowen

### CODE REVIEW

File: ui-v2/app/styles/base/components/buttons/layout.scss [hunk 1]
```diff
@@ -48,3 +48,28 @@
...
+%split-button::before {
+  @extend %as-pseudo;
+  height: 16px;
+}
+%sort-button::before {
+  @extend %with-sort-icon;
+  margin-top: 8px;
+  width: 16px;
+  height: 16px;
+}
```
Comment (johncowen) [inline ui-v2/app/styles/base/components/buttons/layout.scss]: Wondering if there's some stuff that could go in `skin.scss` here, maybe from line 57 down as that is all to do with the pipe and the icon, don't worry too much though we can always hit that in another spring clean PR, or decide when we next chat, its no biggie.

Oh wow I just saw what you did with the `103%`, I'm guessing this is to do with making sure the top and bottom of the pipe meet up with the borders? I love the 103% btw, but I just has a thought that maybe a 'calc(100% + 3px)' might be better (if thats what this is for of course?)

Comment (kaxcode) [reply ui-v2/app/styles/base/components/buttons/layout.scss]: I'll move those lines over to `skin.scss`. 

calc(100% + 3px) is not the same as 103% so it extends too far down. The exact px match would be calc(100% + .5px). Which I think looks worse than the 103%. 🤔

Comment (kaxcode) [reply ui-v2/app/styles/base/components/buttons/layout.scss]: Update on the height. I had the sort matching the search before but I must have undone the change. Adding this change removes the need of a `height: 103%`.

**Comment by kaxcode** [comment]
Future PR needed to improve the performance by having the the sorting happen before the filtering.

**Pull request merged**